specification/inference/delete/DeleteResponse.ts

```diff
@@ -17,8 +17,10 @@
  * under the License.
  */
 
-import { AcknowledgedResponseBase } from '@_types/Base'
+import { DeleteInferenceEndpointResult } from '@inference/_types/Results'
 
 export class Response {
-  body: AcknowledgedResponseBase
+  body: {
+    dryRunResult: DeleteInferenceEndpointResult
```

Comment:
What does `dryRunResult` mean here? Should it be `body: DeleteInferenceEndpointResult` ?